src/config/environment.py

```diff
@@ -0,0 +1,9 @@
+from os import environ as env
+
+DATABASE_CONNECTION = {
+    "dbname": env.get("DB_NAME") or "medgrabber",
```

Comment:
```suggestion
    "dbname": env.get("DB_NAME", "medgrabber"),
```

Dessa forma fica mais pythonico

Reply:
Aí aplica o mesmo pras linhas de baixo também